WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -28,11 +31,16 @@ class WPMainActivityViewModel @Inject constructor() : ViewModel() {
     private val _isBottomSheetShowing = MutableLiveData<Event<Boolean>>()
     val isBottomSheetShowing: LiveData<Event<Boolean>> = _isBottomSheetShowing
 
+    private var mainFabTooltipDisabled = true
```

Comment:
I'd rather not have this state here as a field. Maybe we can create a method `isMainFabTooltipEnabled()` instead? I think it's safe to always go to app prefs to get the value. Keeping this field makes us keep the state in 2 places and increases chance of bugs.

Reply:
The goal was as you guessed to not access the shared every time, but agree maybe it's not worth it in this case and we get the extra plus of having a single state point 👍...changed here 95381cb8653731e40440a61be244ed2eeafd7664